Add SVG label generator tool

jdVotingWorks opened the pull request:
# Overview

## SVG Label Generator Tool

Adds a new Python utility for generating SVG labels with dynamic content.

### What this adds:
- `svg-label-generator/` directory with standalone Python tool
- Generates SVG labels from CSV input data
- Dynamic QR code generation and serial number replacement
- Interactive file selection interface

### Usage:
```bash
cd svg-label-generator
pip install -r requirements.txt
python svg_label_generator.py
```
## Demo Video or Screenshot

## Testing Plan

## Checklist

- [ ] I have prefixed my PR title with "VxDesign: ", "VxPollBook: ", or "HWTA: " if my change is specific to one of those products.
- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate for any new user actions.
- [ ] I have added the "user_facing_change" label to this PR, if relevant, to automate an announcement in #machine-product-updates.

jdVotingWorks committed:
Rewrite: DOM-only SVG edits, no regex

jdVotingWorks committed:
Update README.md

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/votingworks/vxsuite?pullRequest=7061) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/votingworks/vxsuite?pullRequest=7061) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: jdVotingWorks<br/>:x: Jesse DeWald<br/><hr/>**Jesse DeWald** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/votingworks/vxsuite?pullRequest=7061) it.</sub>

jdVotingWorks committed:
scale constant to enlarge qrcode
offset constants to reposition qrcode

jdVotingWorks committed:
updated code to reduce size of label by using a png image instead of paths for the qr codes
removed white backgrounds
removed legacy (and problematic) style flags for text paths

jdVotingWorks committed:
WIP: template generator for making new nameplates

jdVotingWorks committed:
wip: procedurally generated svg templates

jdVotingWorks committed:
WIP: complete refactor based on procedurally generating svgs instead of
     relying on the inkscape -> python workflow
     easier and cleaner to batch the svgs
     and easier to layout for a laser cutter
     need to comb through to make sure AI didn't do anything silly, but seems functional

Commit:
WIP: fixed white fill on template. adjusted offsets

jdVotingWorks committed:
WIP: updated to provide more configuability using the yaml or interactive prompts
     layout seems to work well for both scan and box (which are pretty different reqs)

Commit:
textbox 3 now has the same font size as textbox 2 (and tb3 margin setting is obsolete noop)

Commit:
optional number of lines in textbox2, constrain max font size accordingly

Commit:
filename generation includes laser params

Commit:
adding the yaml as a tracked asset

jdVotingWorks committed:
bb05 and sc14 working configs

Commit:
updated the template generator to work with 1,2,or 3 lines in textbox 2
fixed some wonkyiness in data validation too

Commit:
working template settings for our products (20260219)

jdVotingWorks committed:
updated yaml to reflect new machine

jdVotingWorks committed:
added recursive font searching for multi platform

jdVotingWorks committed:
separating yaml template from live yaml

jdVotingWorks committed:
added yaml template and separated from live yaml (no longer tracked)
moved old code to an archived folder

jdVotingWorks committed:
cleaning up

jdVotingWorks committed:
updated readme and requirements to match the new workflow
